config/cognitoidp/config.go

```diff
@@ -52,5 +54,68 @@ func Configure(p *config.Provider) {
 				"email_verification_subject",
 			},
 		}
+		r.References["email_configuration.configuration_set"] = config.Reference{
+			TerraformName: "aws_ses_configuration_set",
+		}
+		r.References["email_configuration.source_arn"] = config.Reference{
+			TerraformName: "aws_ses_email_identity",
+			Extractor:     common.PathARNExtractor,
+		}
```

Comment:
So it turns out there are actually two terraform resources for these objects, these and the corresponding `aws_sesv2_` ones. As far as I can tell, they are simply two different interfaces for the same objects, with the same ARNs. I'm conflicted about what to do here, but perhaps these would be better left off?

Reply:
It seems more logical not to define a reference for these for now.